packages/insomnia-app/app/ui/components/request-url-bar.js

```diff
@@ -134,13 +134,12 @@ class RequestUrlBar extends React.PureComponent<Props, State> {
       properties: ['openDirectory'],
     };
 
-    remote.dialog.showOpenDialog(options, paths => {
-      if (!paths || paths.length === 0) {
-        return;
-      }
+    const { canceled, filePaths } = await remote.dialog.showOpenDialog(options);
+    if (canceled) {
```

Comment:
Should we be checking for `filePaths.length` here as well, similar to a previous comment?

```suggestion
    if (canceled || !filePaths.length) {
```